parse/table_definitions_arbitrum/balancer/V2_Vault_event_PoolBalanceChanged.json

```diff
@@ -0,0 +1,76 @@
+{
+    "parser": {
+        "abi": {
+            "anonymous": false,
+            "inputs": [
+                {
+                    "indexed": true,
+                    "internalType": "bytes32",
+                    "name": "poolId",
+                    "type": "bytes32"
+                },
+                {
+                    "indexed": true,
+                    "internalType": "address",
+                    "name": "liquidityProvider",
+                    "type": "address"
+                },
+                {
+                    "indexed": false,
+                    "internalType": "contract IERC20[]",
+                    "name": "tokens",
+                    "type": "address[]"
+                },
+                {
+                    "indexed": false,
+                    "internalType": "int256[]",
+                    "name": "deltas",
+                    "type": "int256[]"
+                },
+                {
+                    "indexed": false,
+                    "internalType": "uint256[]",
+                    "name": "protocolFeeAmounts",
+                    "type": "uint256[]"
+                }
+            ],
+            "name": "PoolBalanceChanged",
+            "type": "event"
+        },
+        "contract_address": "0xba12222222228d8ba445958a75a0704d566bf2c8",
+        "field_mapping": {},
+        "type": "log"
+    },
+    "table": {
+        "dataset_name": "balancer",
+        "schema": [
+            {
+                "description": "",
+                "name": "poolId",
+                "type": "STRING"
+            },
+            {
+                "description": "",
+                "name": "liquidityProvider",
+                "type": "STRING"
+            },
+            {
+                "description": "",
+                "name": "tokens",
+                "type": "STRING"
+            },
+            {
+                "description": "",
+                "name": "deltas",
+                "type": "STRING"
+            },
+            {
+                "description": "",
+                "name": "protocolFeeAmounts",
+                "type": "STRING"
+            }
+        ],
+        "table_description": "",
+        "table_name": "Vault_event_PoolBalanceChanged"
```

Comment:
This one also needs to be updated to match the filename. Once this PR merges your next PR should automatically run the checks so you can catch this as soon as you submit a PR.

Reply:
@kome12 Just pushed the fix. If understand correctly, this current PR doesn't have the auto-check functionality enabled? I notice that workflows tend not to update once I push.

Reply:
We've updated the repo settings. The workflows should run now.